main.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+import logging
+import requests
+from concurrent.futures import ThreadPoolExecutor
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+# Dummy data
+dummy_data = [
```

Comment:
Could take this one step further and read in information from local file(s), an input directory for instance.